graphium/trainer/predictor.py

```diff
@@ -447,17 +447,17 @@ def on_train_batch_end(self, outputs, batch: Any, batch_idx: int) -> None:
         tput = num_graphs / train_batch_time
 
         # this code is likely repeated for validation and testing, this should be moved to a function
-        self.task_epoch_summary.update_predictor_state(
-            step_name="train",
-            targets=outputs["targets"],
-            predictions=outputs["preds"],
-            loss=outputs["loss"],  # This is the weighted loss for now, but change to task-specific loss
-            task_losses=outputs["task_losses"],
-            n_epochs=self.current_epoch,
-        )
-        metrics_logs = self.task_epoch_summary.get_metrics_logs()  # Dict[task, metric_logs]
-        metrics_logs["_global"]["grad_norm"] = self.get_gradient_norm()
-        outputs.update(metrics_logs)  # Dict[task, metric_logs]. Concatenate them?
+        # self.task_epoch_summary.update_predictor_state(
+        #     step_name="train",
+        #     targets=outputs["targets"],
+        #     predictions=outputs["preds"],
+        #     loss=outputs["loss"],  # This is the weighted loss for now, but change to task-specific loss
+        #     task_losses=outputs["task_losses"],
+        #     n_epochs=self.current_epoch,
+        # )
+        # metrics_logs = self.task_epoch_summary.get_metrics_logs()  # Dict[task, metric_logs]
+        # metrics_logs["_global"]["grad_norm"] = self.get_gradient_norm()
+        # outputs.update(metrics_logs)  # Dict[task, metric_logs]. Concatenate them?
```

Comment:
Are you sure commenting this out is a good idea?

Reply:
Not sure, I'd like to discuss with @DomInvivo about how to make that change properly, I think it might be useful for something.

Reply:
I think this is slightly different, by using `metrics_every_n_steps`, like we set to 100, we only report the metrcis, including loss every 100 steps, this might not be helpful with the wandb reporting if we wanna track the progress of the training as we only have 157 steps in one epoch. While this change above is about if the "preds" and "targets" are saved each step. I think it would be good to combine the two:
Set `metrics_every_n_steps` to 3 or 5 + remove "preds" and "targets" from step_dict.